test/OpenTelemetry.Exporter.Prometheus.Tests/PrometheusExporterHttpServerTests.cs

```diff
@@ -105,5 +105,44 @@ public async Task PrometheusExporterHttpServerIntegration()
                 "^# TYPE counter_double counter\ncounter_double{key1='value1',key2='value2'} 101.17 \\d+\n$".Replace('\'', '"'),
                 await response.Content.ReadAsStringAsync().ConfigureAwait(false));
         }
+
+        [Theory]
+        [InlineData("http://example.com")]
+        [InlineData("https://example.com")]
+        public void ServerEndpointSanityCheckPositiveTest(string uri)
+        {
+            using MeterProvider meterProvider = Sdk.CreateMeterProviderBuilder()
+                .AddPrometheusExporter(opt =>
+                {
+                    opt.HttpListenerPrefixes = new string[] { uri };
+                })
+                .Build();
+        }
+
+        [Theory]
+        [InlineData("")]
+        [InlineData(null)]
+        [InlineData("ftp://example.com")]
+        public void ServerEndpointSanityCheckNegativeTest(string uri)
+        {
+            try
```

Comment:
Instead of using `try-catch` in tests you can use Assert.Throws, search in this repo for examples

Reply:
Right, the arrange/act/assert is a more standard pattern.
I'll leave the current try-catch so that I can assert the exception message easily.

Reply:
You can still check the exception message using this way.
See [here](https://github.com/open-telemetry/opentelemetry-dotnet/blob/33f3596b4b3b4703daa536e6b5901c554f90a14a/test/OpenTelemetry.Tests/Metrics/MetricViewTests.cs#L64)